core/src/broadcast_stage/broadcast_duplicates_run.rs

```diff
@@ -341,8 +341,12 @@ impl BroadcastRun for BroadcastDuplicatesRun {
         blockstore: &Arc<Blockstore>,
     ) -> Result<()> {
         let (all_shreds, _) = receiver.lock().unwrap().recv()?;
+        let mut shreds = Shreds::default();
+        shreds.shreds = all_shreds.to_vec();
+        // TODO mark timer origin
```

Comment:
Is this TODO already addressed? Might be good to create an issue for outstanding TODO's and reference the issue.